lib/arena.rb

```diff
@@ -1,2 +1,68 @@
+require "pry"
+require_relative "../lib/gladiator"
+
 class Arena
+  attr_accessor :name, :gladiators, :battle, :gladiator_names
+
+  def initialize(name, gladiators = [])
+    @name = name.capitalize
+    @gladiators = gladiators
+    @battle = []
+    @gladiator_names = []
+  end
+
+  def add_gladiator(gladiator)
+    gladiators << gladiator unless gladiators.length == 2
+  end
+
+  def fight
+    unless gladiators.length < 2
+      gladiators.each do |gladiator|
+          battle << gladiator.weapon
+          gladiator_names << gladiator.name
```

Comment:
why not add individual hashes to the `battle` array. So it could be like: 
```ruby
 {name: gladiator.name, weapon: gladiator.weapon}
````
It would change how you iterate through for determining a winner though...you're call